stable/sonatype-nexus/templates/statefulset.yaml

```diff
@@ -108,6 +108,13 @@ spec:
           {{- end }}
         {{- if .Values.nexusBackup.enabled }}
         - name: nexus-backup
+          annotations:
+            {{- range $key, $value := .Values.ingress.annotations }}
+            {{ $key }}: {{ $value | quote }}
+            {{- end }}
+      {{- if .Values.podAnnotations }}
```

Comment:
it looks like you mean `.Values.nexusBackup.podAnnotations` ?? 
and I'm not clear on why you're adding the `ingress.annotations` here either